### PR TITLE
search

### PR DESCRIPTION
**Pull request by prihit**

### CODE REVIEW

File: college/views/public.py [hunk 1]
```diff
@@ -45,3 +45,11 @@ class CollegesView(ListView):
     template_name = 'v2/pages/public/colleges.html'
     context_object_name = 'colleges'
     paginate_by = 20
+
+    def get_queryset(self):
+        name = self.kwargs.get('sea', None)
```
Comment (chayan007) [inline college/views/public.py:50]: use `search`

File: profiles/migrations/0002_auto_20201227_1221.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='otpverification',
             name='is_verified',
-            field=models.SmallIntegerField(default=0),
+            field=models.BooleanField(default=0),
```
Comment (chayan007) [inline profiles/migrations/0002_auto_20201227_1221.py:16]: Please revert the migration.

File: profiles/migrations/0004_auto_20210318_2203.py [hunk 3]
```diff
@@ -13,6 +13,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='otpverification',
             name='is_verified',
-            field=models.SmallIntegerField(default=0),
+            field=models.BooleanField(default=0),
```
Comment (chayan007) [inline profiles/migrations/0004_auto_20210318_2203.py:16]: Please revert the migration.

File: profiles/migrations/0005_auto_20210318_2304.py [hunk 4]
```diff
@@ -13,6 +13,6 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='otpverification',
             name='is_verified',
-            field=models.SmallIntegerField(default=0),
+            field=models.BooleanField(default=0),
```
Comment (chayan007) [inline profiles/migrations/0005_auto_20210318_2304.py:16]: Please revert the migration.

File: college/views/public.py (hunk 1 above)
Comment (chayan007) [reply college/views/public.py:50]: remove `None` -> by default it is `None`

Comment (chayan007) [reply college/views/public.py:50]: Also try this: `self.request.GET.get('search')`

File: profiles/migrations/0004_auto_20210318_2203.py (hunk 3 above)
Comment (prihit) [reply profiles/migrations/0004_auto_20210318_2203.py:16]: What is the command for this ?

File: profiles/migrations/0009_auto_20210404_0208.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.5 on 2021-04-03 20:38
+
```
Comment (chayan007) [inline profiles/migrations/0009_auto_20210404_0208.py:2]: checkout this file @prihit

File: profiles/migrations/0008_merge_20210404_0208.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.1.5 on 2021-04-03 20:38
+
+from django.db import migrations
```
Comment (chayan007) [inline profiles/migrations/0008_merge_20210404_0208.py:3]: checkout this file @prihit

File: profiles/migrations/0009_auto_20210404_0208.py (hunk 5 above)
Comment (prihit) [reply profiles/migrations/0009_auto_20210404_0208.py:2]: image

File: profiles/migrations/0008_merge_20210404_0208.py (hunk 6 above)
Comment (prihit) [reply profiles/migrations/0008_merge_20210404_0208.py:3]: image

Comment (chayan007) [reply profiles/migrations/0008_merge_20210404_0208.py:3]: delete this file then

File: profiles/migrations/0009_auto_20210404_0208.py (hunk 5 above)
Comment (chayan007) [reply profiles/migrations/0009_auto_20210404_0208.py:2]: delete this file then

**Review by chayan007 — APPROVED** [review]

**Pull request merged**